Source/AliveLibCommon/FunctionFwd.hpp

```diff
@@ -7,15 +7,15 @@ bool RunningAsInjectedDll();
 #define CC __cdecl
 #define CCSTD __stdcall
 #if _WIN64
-#define ALIVE_ASSERT_SIZEOF(structureName, expectedSize)
+#define ALIVE_ASSERT_SIZEOF(structureName, expectedSize) static_assert(true)
```

Comment:
is there an more idiomatic way of having a "blank" macro than this?

Reply:
Not really, especially when it's not a function-scope macro. The alternative is something like `struct DUMMY ## __LINE__ { }` which I really do not like.

Leaving the macro completely black does not swallow the semicolon and makes code that forgets the semicolon compile in `_WIN64` mode.